tests/series/test_tensor.py

```diff
@@ -130,7 +130,7 @@ def test_tensor_repr():
         == """╭───────────────────────╮
 │ list_series           │
 │ ---                   │
-│ Tensor(Int64)         │
+│ Tensor[Int64]         │
```

Comment:
Let's keep the old repr, no need for behavior change

Reply:
Addressed in follow-up PR:
https://github.com/Eventual-Inc/Daft/pull/2820